certificates.c

```diff
@@ -135,7 +135,7 @@ hibacert_maybe_inflate(struct sshbuf *extensions) {
 	inflateEnd(&stream);
 
 	if (ret == Z_STREAM_END) {
-		debug2("hibacert_maybe_inflate: using decompressed extension %ld -> %ld bytes", len, sshbuf_len(decomp));
+		debug2("hibacert_maybe_inflate: using decompressed extension %d -> %d bytes", len, sshbuf_len(decomp));
```

Comment:
My bad, but we should actually use %zu for this and the other ones too. %d will cause also failures based as size_t is defined as either long unsigned int or long int based on architectures.

Reply:
Updated. Thanks